### DISCUSSION TITLE
提几个关于游戏打包的问题

### DISCUSSION THREAD

**Discussion by Sunsetra** [Gamuxorg/bbs#341] [游戏或网站运行反馈]
一直在关注贵站的linux游戏，自己也有了贡献一些自己电脑上游戏的想法，不过有些疑问求教：

1. 在Steam上购买的游戏能否进行打包呢？会不会出现打包后无法在别的电脑（无Steam环境）上运行的问题？

2. 我用的发行版是Archlinux，正如贵站打包说明所说，系统更新比较频繁，依赖库版本可能比较新，在这样的系统上打包的游戏会不会有跨发行版的兼容性问题？（在我的系统上能完美运行）

3. 打包说明文档中提到了function.sh这个文件，但说明文档中提供下载的百度盘链接已经失效了，我在AppImage的Github页只看到了build.sh文件，这两个是同一个文件吗？如果不是，烦请更新一下可用的下载链接。

本人很看好贵站的宗旨，希望能贡献自己的力量扩展贵站的游戏库。

**Comment by shouhuanxiaoji** [comment]
steam上买的游戏可以打包，不过要先crack。其实gog和humble store的比较好。
如果你是买的steam的游戏，打包appimage根本不需要function.sh或build.sh，这些脚本是辅助处理依赖的，但是商业游戏一般都没有依赖问题，在各种发行版兼容都比较好，可以直接打包。
steam游戏打包比较简单，套用固定模板就可以，你可以加群274328087，我们聊一下。
